test/units/TEST-35-LOGIN.sh

```diff
@@ -368,7 +368,7 @@ testcase_sanity_check() {
 
     # Run most of the loginctl commands from a user session to make
     # the seat/session autodetection work-ish
-    systemd-run --user --pipe --wait -M "testuser@.host" bash -eux <<\EOF
+    systemd-run --setenv=SYSTEMD_PAGER --user --pipe --wait -M "testuser@.host" bash -eux <<\EOF
```

Comment:
> Otherwise loginctl user-status will get stuck in a pager when running interactively.

Why the command stuck ? Is it something should be fixed in loginctl ?

Reply:
That's why I had systemd-tty-ask-password-agent running:

```
    594 hvc0     Ss+    0:00 bash /usr/lib/systemd/tests/testdata/units/TEST-35-LOGIN.sh
    885 hvc0     S+     0:00  \_ bash /usr/lib/systemd/tests/testdata/units/TEST-35-LOGIN.sh
    927 hvc0     S+     0:00      \_ systemctl -T --wait restart getty@tty2.service
    928 hvc0     S+     0:00          \_ /usr/bin/systemd-tty-ask-password-agent --watch
```